src/commands/play.rs

```diff
@@ -133,14 +147,13 @@ async fn calculate_time_until_play(queue: &[TrackHandle], flag: &PlayFlag) -> Op
 
             Some(durations + top_track_duration - top_track_elapsed)
         }
-        PlayFlag::PLAYTOP => Some(top_track_duration - top_track_elapsed),
     }
 }
 
 async fn enqueue_playlist(call: &Arc<Mutex<Call>>, uri: &str) {
     if let Some(urls) = YouTubeRestartable::ytdl_playlist(uri).await {
         for url in urls.iter() {
-            enqueue_song(call, url.to_string(), true, &PlayFlag::DEFAULT).await;
+            enqueue_song(call, url.to_string(), true, &PlayFlag::END).await;
```

Comment:
Change `enqueue_song` too to take a owned value of the play flag instead of a reference.